src/OrchardCore.Build/Dependencies.AspNetCore.props

```diff
@@ -1,10 +1,32 @@
 <Project>
   <PropertyGroup>
-    <AspNetCoreVersion>3.1.11</AspNetCoreVersion>
-    <AspNetCoreTargetFramework>netcoreapp3.1</AspNetCoreTargetFramework>
+    <AspNetCoreTargetFrameworks>netcoreapp3.1;net5.0</AspNetCoreTargetFrameworks>
+    <TargetFrameworks>$(AspNetCoreTargetFrameworks)</TargetFrameworks>
   </PropertyGroup>
+  <Choose>
+    <When Condition=" '$(TargetFramework)' == 'net5.0' ">
+      <PropertyGroup>
+        <AspNetCoreVersion>5.0.1</AspNetCoreVersion>
```

Comment:
```suggestion
        <AspNetCoreVersion>5.0.3</AspNetCoreVersion>
```

Reply:
@sebastienros Thanks, I'll check that out.  I had specifically set it to 5.0.2 before and had some issues with packages not being found.  I'll try out 5.0.3. :)

Reply:
Done